packages/redux/src/store.ts

```diff
@@ -59,6 +59,7 @@ export interface WorkspaceSettings {
   };
   webgl: boolean;
   webglEditable: boolean;
+  gridDensity?: 'compact' | 'normal' | 'spacious';
```

Comment:
This should always be set. Also we may as well match the Spectrum density props, which use `regular` instead of `normal` (though not every component has a `spacious` option).
```suggestion
  gridDensity: 'compact' | 'regular' | 'spacious';
```
Then define what the default value is in `LocalWorkspaceStorage`.
In the `updateWorkspaceData` action, we map the users `CustomizableWorkspaceSettings` on top of the default settings. That way the default values are always set in the `WorkspaceSettings` we get from redux, and there's no ambiguity on what the "default" value is.

Side note, we should probably allow this to be set via a prop as well (for a default value). Can be a separate ticket.

Reply:
Ya I initially made it optional since it wouldn't be set in enterprise right away, but shouldn't matter until we add the settings menu in enterprise

Reply:
Would be nice to add to both but no urgency for DHC